src/scripts/webpack/webpack.base.js

```diff
@@ -55,7 +56,9 @@ export default ({ config, paths, NODE_ENV, jsLoader = JSLoader('babel') }) => {
         syntax: 'scss'
       }),
       new HTMLPlugin(getHtmlPluginConfig(NODE_ENV, config, paths))
-    ],
+    ].concat(
+      bundleAnalyzer ? [new BundleAnalyzerPlugin({ generateStatsFile: true })] : []
```

Comment:
why `generateStatsFile: true`?

Reply:
Some additional analyzer tools require the stats file upload to be used, such as:
* https://chrisbateman.github.io/webpack-visualizer/
* https://webpack.github.io/analyse/
* https://alexkuz.github.io/webpack-chart/

Reply:
Right, let's not speculate, I removed it and we will add it if someone actually needs it in the future 😛